tests/test_api.py

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2023 Collabora Limited
+# Author: Guillaume Tucker <guillaume.tucker@collabora.com>
+
+"""Unit tests for KernelCI API bindings"""
+
+import kernelci.api
+import kernelci.api.helper
+import kernelci.config
+
+
+def test_api_init():
+    """Test that all the API configurations can be initialised (offline)"""
+    config = kernelci.config.load('tests/configs/api-configs.yaml')
+    api_configs = config['api_configs']
+    for api_name, api_config in api_configs.items():
+        print(f"API config name: {api_name}")
+        api = kernelci.api.get_api(api_config)
+        assert api is not None
+        kernelci.api.helper.APIHelper(api)
```

Comment:
Shouldn't we have an assertion for `APIHelper` instance being created?

Reply:
If it fails then an exception is raised which would cause the test to fail.  We can assert that the object is not `None` I guess...

Reply:
After discussing this, I'll add an assert that the created object is of the type `APIHelper`.  We'll still have an exception if the contructor fails, but then we'll also have an explicit test to check the object is of the expected base type.

Reply:
Updated with asserts using `isinstance()`.